### ISSUE TITLE
got error in collapsible-header-with-tabview 

### ISSUE THREAD

**Issue by imansalhi** [react-native-community/react-native-tab-view#575]
I followed [this example](https://snack.expo.io/@satya164/collapsible-header-with-tabview) about collapsible-header-with-tabview but got this error at run:

```
 Invariant Violation: Invariant Violation: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.
 ReactNativeJS: Check the render method of `TabView`.`
```

**Comment by imansalhi** [comment]
I think i khew what's the problem...the code has an import like this:
`import { TabViewAnimated, TabBar } from 'react-native-tab-view';`
but there isn't any  `TabViewAnimated` class in  `react-native-tab-view` .
i think my `react-native-tab-view` version in **1.0**

**Comment by satya164** [comment]
Hey, I just released a new alpha `2.0.0-alpha.0` of the library. It's rewritten using [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler) and [react-native-reanimated](https://github.com/kmagiera/react-native-reanimated) addresses a many platform specific bugs and performance problems. The documentation is updated as well.

Please try the new version and see if it addresses your issue. If not, please open a new issue following the issue template.
